app-info/eos-extra.xml

```diff
@@ -30,7 +30,7 @@
     <kudos>
       <kudo>GnomeSoftware::popular</kudo>
     </kudos>
-    <bundle type="flatpak"></bundle>
+    <bundle type="flatpak">app/com.calibre_ebook.calibre//stable</bundle>
   </component>
   <component type="desktop" merge="append">
     <id>com.dropbox.Client.desktop</id>
```

Comment:
What about the `<bundle>` for this one?

Reply:
Dropbox is not marked as popular as of now. Should we mark it popular ?

Reply:
I overlooked that it wasn't marked as popular. OTOH, if these categories are standard maybe we should add them to flathub/com.dropbox.Client? Perhaps draining these is one for a separate task.

Reply:
Yes, that's what I was thinking. There is stuff in `eos-extra.xml` that needs to revisited. I was "just" focused on the popular / app kudo thing lately. Revisiting and actually fixing the eos-extra list is what we need to plan into a separate task.